htdocs/libraries/icms/core/Object.php

```diff
@@ -437,7 +437,7 @@ public function getVar($key, $format = 's') {
 				break;
 
 			case XOBJ_DTYPE_ARRAY:
-				$ret = unserialize($ret);
+				$ret = unserialize($ret) ? unserialize($ret) : array();
```

Comment:
`unserialize` sometimes can throw exceptions, so I think this is safer approach:
```suggestion
	try {
        $ret = unserialize($data);
        if ($ret === false) {
            $ret = [];
        }
    } catch (\Throwable $e) {
        $ret = [];
    }
```